source/tests/system/nidaqmx_driver_api_tests.cpp

```diff
@@ -1362,21 +1372,27 @@ TEST_F(NiDAQmxDriverApiTests, CalculateReversePolyCoefficientsWithPositiveRevers
 TEST_F(NiDAQmxDriverApiTests, AIVoltageChannel_WaitForValidTimestamp_ReturnsError)
 {
   create_ai_voltage_chan(0.0, 1.0);
```

Comment:
It looks like these tests are organized as arrange/act/assert. You probably ought to keep the spacing, to the extent possible.

Reply:
I did the best I could. In some cases especially where there are 2 calls to close with mixed success/failure, it's hard now.